server/schema/TypeDefs.ts

```diff
@@ -93,6 +95,7 @@ const typeDefs = gql`
         time: Int
```

Comment:
This is still `time` should it be timestamp?

Reply:
changed